modules/cnf-verify-sriov-operator-n3000.adoc

```diff
@@ -0,0 +1,251 @@
+
+.Prerequisites
+
+* Intel® FPGA PAC N3000 card (Optional)
+* Node or nodes installed with the n3000-operator
+* Node or nodes installed with the SR-IOV-FEC operator
+* Real-Time kernel and huge pages configured with Performance Addon Operator
+
+.Procedure
+
+. Create a namespace for the test by completing the following actions:
+
+.. Define the `test-bbdev` namespace by creating a file named `test-bbdev-namespace.yaml` file as shown in the following example:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: test-bbdev
+  labels:
+    openshift.io/run-level: "1"
+----
+
+.. Create the namespace by running the following command:
++
+[source,terminal]
+----
+$  oc create -f `test-bbdev-namespace.yaml`
+----
+
+. Create the following `Pod` specification, and then save the YAML in the `pod-test.yaml` file:
++
+[source,yaml]
+----
+    apiVersion: v1
+    kind: Pod
+    metadata:
+      name: pod-bbdev-sample-app
+      namespace: test-bbdev <1>
+    spec:
+      containers:
+      - securityContext:
+          privileged: false
+          capabilities:
+            add:
+              - IPC_LOCK
+              - SYS_NICE
+        name: bbdev-sample-app
+        image: bbdev-sample-app:1.0  <2>
+        command: [ "sudo", "/bin/bash", "-c", "--" ]
+        runAsUser: 0 <3>
+        resources:
+          requests:
+            hugepages-1Gi: 4Gi <4>
+            memory: 1Gi
+            cpu: "4" <5>
+            intel.com/intel_fec_5g: '1' <6>
+            #intel.com/intel_fec_acc100: '1'
+            #intel.com/intel_fec_lte: '1'
+          limits:
+            memory: 4Gi
+            cpu: "4"
+            hugepages-1Gi: 4Gi
+            intel.com/intel_fec_5g: '1'
+            #intel.com/intel_fec_acc100: '1'
+            #intel.com/intel_fec_lte: '1'
+----
+<1> Specify the `namespace` you created in step 1.
+<2> This defines the test image containing the compiled DPDK.
+<3> Make the container execute internally as the root user.
+<4> Specify hugepage size hugepages-1Gi and the quantity of hugepages that will be allocated to the pod. Hugepages and isolated CPUs needs to be configure using the Performance Addon Operator.
+<5> Specify the number of CPUs.
+<6> Testing of the N3000 5G FEC configuration is supported by `intel.com/intel_fec_5g`.
++
+[NOTE]
+====
+To test the ACC100 configuration, uncomment `intel.com/intel_fec_acc100` by removing the # symbol.
+To test the N3000 4G/LTE configuration, uncomment `intel.com/intel_fec_lte` by removing the # symbol.
+Only one resource should be active at any time.
+====
+
+. Create the pod:
++
+[source,terminal]
+----
+$  oc apply -f pod-test.yaml
+----
+
+. Check that the pod is created:
++
+[source,terminal]
+----
+$  oc get pods
+----
++
+.Example output
+[source,terminal]
+----
+NAME                                            READY           STATUS          RESTARTS        AGE
+fec-pod                                         1/1             Running         0               10s
```

Comment:
the pod name should be `pod-bbdev-sample-app` not `fec-pod` according to` pod-test.yaml`